design-tokens-manager/src/routes/Scaler/ControlPane.jsx

```diff
@@ -0,0 +1,125 @@
+import {
+  Box,
+  Button,
+  Form,
+  FormField,
+  Heading,
+  RangeInput,
+  Select,
+  Text,
+} from 'grommet';
+
+const RangeWrapper = ({ children, max, min }) => {
+  return (
+    <Box
+      direction="row"
+      gap="small"
+      align="center"
+      justify="between"
+      margin={{ horizontal: 'xsmall', vertical: 'xxsmall' }}
+    >
+      <Text size="small" weight={500}>
+        {min}
+      </Text>
+      {children}
+      <Text size="small" weight={500}>
+        {max}
+      </Text>
+    </Box>
+  );
+};
+
+export const ControlPane = ({ defaultValues, values, setValues, ...rest }) => {
+  const onChange = nextValues => {
+    setValues(nextValues);
+  };
+
+  const BASE_OPTIONS = [4, 6, 8, 12, 16, 18, 24];
+  const FACTOR_OPTIONS = [1.2, 1.25, 1.333, 1.414, 1.5, 1.618, 2];
+  const STEP_OPTIONS = [7, 8, 9, 10, 11, 12, 13, 14, 15];
+
+  return (
+    <Box
+      background="background-front"
+      flex={false}
+      gap="medium"
+      pad="medium"
+      round="medium"
+      {...rest}
+    >
+      <Heading level={2} margin="none">
+        Scale settings
+      </Heading>
+      <Box width={{ min: 'small', max: 'medium' }}>
+        <Form value={values} onChange={onChange}>
+          <Box width={{ min: 'xsmall', max: 'small' }}>
+            <FormField
+              htmlFor="base"
+              name="base"
+              label="Base unit"
+              help="Foundation unit for the scale"
+            >
+              <Select id="base" name="base" options={BASE_OPTIONS} />
```

Comment:
```suggestion
              <Select id="base__input" name="base" options={BASE_OPTIONS} />
```
More of an accessibility detail, but do we need this? Or is label getting associated without this already?

Reply:
** Or actually this should be commented above on the htmlFor line instead of on this id

Reply:
Good catch. Fixing on another branch.